src/components/Personalization/index.js

```diff
@@ -10,14 +10,31 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { isNonEmptyArray } from "../../utils";
+import { isNonEmptyArray, uuid, getTopLevelCookieDomain } from "../../utils";
 import { initRuleComponentModules, executeRules } from "./turbine";
 import { hideContainers, showContainers, hideElements } from "./flicker";
 
 const PAGE_HANDLE = "personalization:page";
+const SESSION_ID_COOKIE = "alloy-session-id";
+const SESSION_ID_TTL = 31 * 60 * 1000;
```

Comment:
31 minutes? What the reasoning behind that value? Just curious more than anything.

Reply:
This is what we currently have in Adobe Target, basically if there is 31 mins of inactivity.